spec/jobs/search_keyword_job_spec.rb

```diff
@@ -8,14 +8,77 @@
   describe '#perform' do
     context 'given a valid request' do
       it 'updates the keyword status as completed', vcr: 'services/search/google/valid' do
-        search_result = SearchResult.create(keyword: 'ruby', search_engine: 'google')
+        search_result = SearchResult.create(keyword: 'tivi', search_engine: 'google')
 
         expect(search_result.status).to eq('in_progress')
 
         expect { described_class.perform_now(search_result.id) }
           .to change { search_result.reload.status }
           .to('completed').and change { search_result.html_code }
-          .to(include('<title>ruby - Google Search</title>'))
+          .to(include('<title>tivi - Google Search</title>'))
+      end
+
+      it 'updates the adwords_top_urls correctly', vcr: 'services/search/google/valid' do
```

Comment:
We can omit the adverb `correctly` 😅 

```suggestion
      it 'updates the adwords_top_urls with the parsed content from the seach page', vcr: 'services/search/google/valid' do
```